docs/about/05_archetype.adoc

```diff
@@ -0,0 +1,64 @@
+
+== Archetypes
+
+Helidon provides the following set of archetypes to bootstrap your application development journey.
+
+=== Bare
+This option creates a minimal Helidon project, suitable to start from scratch. It includes only required dependencies e.g. in case of Helidon MP, it uses `helidon-microprofile-core` bundle instead of `helidon-microprofile` bundle and adds required dependencies to it. It also doesn't have out-of-box observability setup.
+
+=== QuickStart
+This option builds on `Bare` to include multiple REST operations along with default observability setup and a set of additional dependencies to enable ease of development e.g. in case of Helidon MP, it uses `helidon-microprofile` bundle instead of `helidon-microprofile-core` bundle.
+
+=== Database
+This option builds on `QuickStart` to demonstrate how to integrate with database (in-memory H2, by default). In case of, Helidon SE that uses the DbClient API while for Helidon MP that uses JPA.
+
+=== OCI
+This option builds on `QuickStart` to demonstrate integration with Oracle Cloud Infrastructure (OCI) services using the OCI SDK. Generated project showcases OpenApi-driven development approach where the practice of designing and building APIs is done first,
+then creating the rest of an application around them is implemented next. This is available for Helidon MP only.
+
+== Generated Application Structure
+
+User can scaffold a new Maven project based on these archetypes. See <<./05_cli.adoc, Helidon CLI>> and our <<se/guides/02_quickstart.adoc, Helidon SE QuickStart Guide>> or <<mp/guides/02_quickstart.adoc, Helidon MP QuickStart Guide>> for details on usage.
+
+Once the archetype is selected the other options have defaults and the project is generated in a directory named after the `artifactId` value. It mainly contains the following:
+
+- Maven structure
+- skeletal application code
+- associated unit test code
+- example Dockerfile files
+- application configuration file(s)
+- instructions to build and run application/test
+
+== Using Generated Application
+
+The easiest way to get started is follow instructions in README file and familiarize with layout and features provided to build upon them.
+
+Also look at the pom.xml. You will find the suitable Helidon parent pom, allowing you to omit the version of the different Helidon dependencies. The parent pom also brings in helidon-maven-plugin responsible for packaging of the application and providing the development mode.
+
+Helidon recommends to utilize Jandex at build time esp. for MP. One immediately recognizable application of this is CDI bean discovery i.e. to discover various classes or annotations. It is also required by `smallrye` that we use for OpenAPI integration. This index can be created by `jandex-maven-plugin` plugin which is provided with Helidon parent pom.
```

Comment:
Jandex is only used in MP, as `jandex-maven-plugin` is configured in `io.helidon.application:helidon-mp`.

Reply:
Removed two paragraph as suggested in other comment.